### ISSUE TITLE
rest api rison bug if send an invalid Rison argument

### ISSUE THREAD

**Issue by wukuan405** [dpgaspar/Flask-AppBuilder#1177]
### Environment
pip freeze output:
```
Flask==1.1.1
Flask-Admin==1.5.3
Flask-AppBuilder==2.2.0
Flask-Babel==0.12.2
Flask-Bcrypt==0.7.1
Flask-Caching==1.3.3
Flask-HTTPAuth==3.3.0
Flask-JWT-Extended==3.24.1
Flask-Login==0.4.1
Flask-OpenID==1.2.5
Flask-RESTful==0.3.7
flask-shell==0.1.3
Flask-SQLAlchemy==2.4.1
flask-swagger==0.2.13
Flask-Testing==0.7.1
Flask-WTF==0.14.2
```
### Steps to reproduce
```bash
$ curl -i 'http://localhost:8080/api/v1/example/risonjson?q=('
or
$ curl -i 'http://localhost:8080/api/v1/example/greeting4?q=('
```

### Describe the actual results
```
RecursionError: maximum recursion depth exceeded in comparison

Content-Type: text/html; charset=utf-8
X-XSS-Protection: 0
Connection: close
Server: Werkzeug/0.16.0 Python/3.6.8
Date: Sun, 03 Nov 2019 00:33:38 GMT

<!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 4.01 Transitional//EN"
  "http://www.w3.org/TR/html4/loose.dtd">
<html>
  <head>
    <title>RecursionError: maximum recursion depth exceeded in comparison // Werkzeug Debugger</title>
```

**Comment by dpgaspar** [comment]
Thank you for reporting, curious behaviour. I'll investigate

**Comment by dpgaspar** [comment]
Should be included in 2.3.2
